core/controller/src/main/scala/whisk/core/entitlement/ActivationThrottler.scala

```diff
@@ -39,11 +40,11 @@ class ActivationThrottler(loadBalancer: LoadBalancer, defaultConcurrencyLimit: I
     /**
      * Checks whether the operation should be allowed to proceed.
      */
-    def check(user: Identity)(implicit tid: TransactionId): Boolean = {
+    def check(user: Identity)(implicit tid: TransactionId): RateLimit = {
         val concurrentActivations = loadBalancer.activeActivationsFor(user.uuid)
         val concurrencyLimit = user.limits.concurrentInvocations.getOrElse(defaultConcurrencyLimit)
-        logging.info(this, s"namespace = ${user.uuid.asString}, concurrent activations = $concurrentActivations, below limit = $concurrencyLimit")
-        concurrentActivations < concurrencyLimit
+        logging.info(this, s"namespace = ${user.uuid.asString}, concurrent activations = $concurrentActivations, limit = $concurrencyLimit")
```

Comment:
for easier log searching, having the user namespace string included with the namespace uuid would be nice.